news-aggregation/news-sources/TechCrunchNewsSource.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    IRead,
+    IModify,
+    IHttp,
+    IPersistence,
+} from "@rocket.chat/apps-engine/definition/accessors";
+import { NewsAggregationApp } from "../NewsAggregationApp";
+import { NewsItem } from "../definitions/NewsItem";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+
+export class NewsSource {
+    news: NewsItem[] = [];
+
+    constructor(app: NewsAggregationApp, news: NewsItem[]) {
+        this.news = news;
+    }
+
+    async fetchNews(
+        read: IRead,
+        modify: IModify,
+        room: IRoom,
+        http: IHttp,
+        persis: IPersistence
+    ): Promise<any> {}
+
+    async getNews(
+        read: IRead,
+        modify: IModify,
+        room: IRoom,
+        http: IHttp,
+        persis: IPersistence
+    ): Promise<any> {}
+}
```

Comment:
This part should be removed. And the `NewsSource` class from `news-aggregation/definitions/NewsSource.ts` should be used.